src/main.rs

```
@@ -144,26 +109,26 @@ fn real_main(options: Options, config: &Config) -> CliResult {
     let ebuild_path = PathBuf::from(format!("{}-{}.ebuild", package.name(), package.version()));
 
     // Open the file where we'll write the ebuild
-    let mut file = try!(OpenOptions::new()
-                            .write(true)
-                            .create(true)
-                            .truncate(true)
-                            .open(&ebuild_path)
-                            .chain_err(|| "failed to create ebuild"));
+    let mut file = OpenOptions::new()
+        .write(true)
+        .create(true)
+        .truncate(true)
+        .open(&ebuild_path)
+        .unwrap();
 
     // write the contents out
-    try!(write!(file,
-                include_str!("ebuild.template"),
-                description = desc.trim(),
-                homepage = homepage.trim(),
-                license = license.trim(),
-                crates = crates.join(""),
-                cargo_ebuild_ver = env!("CARGO_PKG_VERSION"),
-                this_year = 1900 + time::now().tm_year,
-                ).chain_err(|| "unable to write ebuild to disk"));
+    write!(
+        file,
+        include_str!("ebuild.template"),
+        description = desc.trim(),
+        homepage = homepage.trim(),
+        license = license.trim(),
+        crates = crates.join(""),
+        cargo_ebuild_ver = env!("CARGO_PKG_VERSION"),
+        this_year = 1900 + time::now().tm_year,
+    ).expect("Error during ebuild file writing");
```

Again this is going to panic instead of returning a nice error to the user.